modules/auxiliary/gather/browser_info.rb

```diff
@@ -74,7 +74,7 @@ def print_target_info(cli, target_info)
 
   def on_request_exploit(cli, req, target_info)
     print_target_info(cli, target_info)
-    send_not_found(cli)
+    send_response(cli, '')
```

Comment:
Sorry, don't have framework code on my phone to grok this, could you explain why this change is needed? If it's a hack around lib code, let's fix lib.